phpunit.xml.dist

```diff
@@ -0,0 +1,17 @@
+<phpunit bootstrap="vendor/autoload.php">
```

Comment:
@jbout What do u think adding this file on each extensions? as @Skyld said: ```the xml itself is in gitignore,
the dist file serves as a bluprint for the configuration with the minimal settings```. Can also include report for extension.